SonarQube.Client.Tests/SonarQubeService_GetHotspotAsync.cs

```diff
@@ -60,8 +60,13 @@ public async Task GetHotspot_Response_From_SonarQube()
     ""securityCategory"": ""others"",
     ""vulnerabilityProbability"": ""LOW""
   },
+  ""textRange"": {
```

Comment:
Hhmm; interesting. This element doesn't appear in the example response in the webapi documentation for either 'next' (v8.6) or 'peach' (v8.5) (which is where the example used in the test came from). Is it just the webapi documentation that is wrong?

Also, assuming that the product code does now return the textRange, do we need to update the minimum version in our client code in which this API is supported? (currently listed as v8.1).

Reply:
Good question; I took the response directly from the network, so that's what's being returned from Next at the moment.
I'll ask SQ folks regarding the docs + version.